androidHackathonApp/src/main/AndroidManifest.xml

```diff
@@ -18,7 +18,7 @@
             </intent-filter>
         </activity>
         <activity
```

Comment:
Can probably move this into the manifest for the module because it merges them automatically